akka-http-core/src/main/resources/reference.conf

```diff
@@ -135,8 +135,11 @@ akka.http {
     # doesn't have to be fiddled with in most applications.
     request-header-size-hint = 512
 
-    # The proxy configurations to be used for requests with the specified
-    # scheme.
+    # The proxy configurations to be used for requests with the specified scheme.
+    # !! WARNING !!
+    # Client proxying is currently not implemented. Refer to https://github.com/akka/akka/issues/16853 for status.
+    # Community help on implementing this feature would be very welcome.
```

Comment:
removing the options is a bad idea, and someone just complained that this made them think it should work: https://github.com/akka/akka/issues/16853#issuecomment-242320860 I agree that if I see such config I'd expect it does something